src/main/java/seedu/address/storage/JsonAdaptedStatus.java

```diff
@@ -0,0 +1,52 @@
+package seedu.address.storage;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.model.status.Status;
+
+/**
+ * Jackson-friendly version of {@link Status}.
+ */
+class JsonAdaptedStatus {
+
+    private final String status;
+
+    /**
+     * Constructs a {@code JsonAdaptedTag} with the given {@code status}.
+     */
+    @JsonCreator
+    public JsonAdaptedStatus(String status) {
+        this.status = status;
+    }
+
+    /**
+     * Converts a given {@code Status} into this class for Jackson use.
```

Comment:
```suggestion
     * Converts a given {@code Status} into a {@code JsonAdaptedStatus} for Jackson use.
```
This works better for method hints when using this method in other classes.

Reply:
Hi I have resolved this thanks.